CRITICAL - 2022-02-16 03:46:41 --> Call to undefined function set_value()
#0 D:\xammp\htdocs\ci4\system\View\View.php(210): include()
#1 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('registration', Array, true)
#3 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(14): view('registration', Array)
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->index()
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 03:49:17 --> include(include/head.php): failed to open stream: No such file or directory
#0 D:\xammp\htdocs\ci4\app\Views\login.php(2): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(include...', 'D:\\xammp\\htdocs...', 2, Array)
#1 D:\xammp\htdocs\ci4\app\Views\login.php(2): include()
#2 D:\xammp\htdocs\ci4\system\View\View.php(210): include('D:\\xammp\\htdocs...')
#3 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login.php', Array, true)
#5 D:\xammp\htdocs\ci4\app\Controllers\Login.php(13): view('login.php', Array)
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Login->index()
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-02-16 03:49:17 --> Call to undefined function set_value()
#0 D:\xammp\htdocs\ci4\system\View\View.php(210): include()
#1 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('registration', Array, true)
#3 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(14): view('registration', Array)
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->index()
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 03:49:29 --> include(include/head.php): failed to open stream: No such file or directory
#0 D:\xammp\htdocs\ci4\app\Views\login.php(2): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(include...', 'D:\\xammp\\htdocs...', 2, Array)
#1 D:\xammp\htdocs\ci4\app\Views\login.php(2): include()
#2 D:\xammp\htdocs\ci4\system\View\View.php(210): include('D:\\xammp\\htdocs...')
#3 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login.php', Array, true)
#5 D:\xammp\htdocs\ci4\app\Controllers\Login.php(13): view('login.php', Array)
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Login->index()
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-02-16 03:49:31 --> Call to undefined function set_value()
#0 D:\xammp\htdocs\ci4\system\View\View.php(210): include()
#1 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('registration', Array, true)
#3 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(14): view('registration', Array)
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->index()
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 03:49:33 --> Call to undefined function set_value()
#0 D:\xammp\htdocs\ci4\system\View\View.php(210): include()
#1 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('registration', Array, true)
#3 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(14): view('registration', Array)
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->index()
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 03:50:04 --> Call to undefined function set_value()
#0 D:\xammp\htdocs\ci4\system\View\View.php(210): include()
#1 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('registration', Array, true)
#3 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(14): view('registration', Array)
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->index()
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 03:50:06 --> Call to undefined function set_value()
#0 D:\xammp\htdocs\ci4\system\View\View.php(210): include()
#1 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('registration', Array, true)
#3 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(14): view('registration', Array)
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->index()
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 03:50:21 --> Call to undefined function set_value()
#0 D:\xammp\htdocs\ci4\system\View\View.php(210): include()
#1 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('registration', Array, true)
#3 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(14): view('registration', Array)
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->index()
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 03:50:49 --> Call to undefined function set_value()
#0 D:\xammp\htdocs\ci4\system\View\View.php(210): include()
#1 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('registration', Array, true)
#3 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(14): view('registration', Array)
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->index()
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 03:50:50 --> Call to undefined function set_value()
#0 D:\xammp\htdocs\ci4\system\View\View.php(210): include()
#1 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('registration', Array, true)
#3 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(14): view('registration', Array)
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->index()
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 03:51:02 --> Call to undefined function set_value()
#0 D:\xammp\htdocs\ci4\system\View\View.php(210): include()
#1 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('registration', Array, true)
#3 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(14): view('registration', Array)
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->index()
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 05:35:41 --> include(include/head.php): failed to open stream: No such file or directory
#0 D:\xammp\htdocs\ci4\app\Views\login.php(2): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(include...', 'D:\\xammp\\htdocs...', 2, Array)
#1 D:\xammp\htdocs\ci4\app\Views\login.php(2): include()
#2 D:\xammp\htdocs\ci4\system\View\View.php(210): include('D:\\xammp\\htdocs...')
#3 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login.php', Array, true)
#5 D:\xammp\htdocs\ci4\app\Controllers\Login.php(13): view('login.php', Array)
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Login->index()
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-02-16 08:56:12 --> Call to undefined function set_value()
#0 D:\xammp\htdocs\ci4\system\View\View.php(210): include()
#1 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('registration', Array, true)
#3 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(14): view('registration', Array)
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->index()
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 08:56:15 --> include(include/head.php): failed to open stream: No such file or directory
#0 D:\xammp\htdocs\ci4\app\Views\login.php(2): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(include...', 'D:\\xammp\\htdocs...', 2, Array)
#1 D:\xammp\htdocs\ci4\app\Views\login.php(2): include()
#2 D:\xammp\htdocs\ci4\system\View\View.php(210): include('D:\\xammp\\htdocs...')
#3 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login.php', Array, true)
#5 D:\xammp\htdocs\ci4\app\Controllers\Login.php(13): view('login.php', Array)
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Login->index()
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-02-16 08:56:17 --> Call to undefined function set_value()
#0 D:\xammp\htdocs\ci4\system\View\View.php(210): include()
#1 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('registration', Array, true)
#3 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(14): view('registration', Array)
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->index()
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 08:56:28 --> Call to undefined function set_value()
#0 D:\xammp\htdocs\ci4\system\View\View.php(210): include()
#1 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('registration', Array, true)
#3 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(14): view('registration', Array)
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->index()
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 09:03:06 --> Call to undefined function set_value()
#0 D:\xammp\htdocs\ci4\system\View\View.php(210): include()
#1 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('registration', Array, true)
#3 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(14): view('registration', Array)
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->index()
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 09:03:07 --> Call to undefined function set_value()
#0 D:\xammp\htdocs\ci4\system\View\View.php(210): include()
#1 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('registration', Array, true)
#3 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(14): view('registration', Array)
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->index()
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 09:04:21 --> Call to undefined function set_value()
#0 D:\xammp\htdocs\ci4\system\View\View.php(210): include()
#1 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('registration', Array, true)
#3 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(14): view('registration', Array)
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->index()
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 09:04:31 --> Call to undefined function set_value()
#0 D:\xammp\htdocs\ci4\system\View\View.php(210): include()
#1 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('registration', Array, true)
#3 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(14): view('registration', Array)
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->index()
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 09:06:33 --> Call to undefined function App\Controllers\validate()
#0 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->registerCustomer()
#1 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#2 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2022-02-16 09:06:58 --> Call to undefined function App\Controllers\validate()
#0 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->registerCustomer()
#1 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#2 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2022-02-16 09:07:30 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 09:07:32 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 09:07:33 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 09:07:36 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 09:07:37 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 09:07:39 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 10:56:23 --> include(include/head.php): failed to open stream: No such file or directory
#0 D:\xammp\htdocs\ci4\app\Views\login.php(2): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(include...', 'D:\\xammp\\htdocs...', 2, Array)
#1 D:\xammp\htdocs\ci4\app\Views\login.php(2): include()
#2 D:\xammp\htdocs\ci4\system\View\View.php(210): include('D:\\xammp\\htdocs...')
#3 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login', Array, true)
#5 D:\xammp\htdocs\ci4\app\Config\Routes.php(41): view('login')
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(781): CodeIgniter\CodeIgniter->Config\{closure}()
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-02-16 10:59:46 --> Undefined property: App\Controllers\Login::$load
#0 D:\xammp\htdocs\ci4\app\Controllers\Login.php(10): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'D:\\xammp\\htdocs...', 10, Array)
#1 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Login->index()
#2 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-02-16 11:00:10 --> include(include/head.php): failed to open stream: No such file or directory
#0 D:\xammp\htdocs\ci4\app\Views\login.php(2): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(include...', 'D:\\xammp\\htdocs...', 2, Array)
#1 D:\xammp\htdocs\ci4\app\Views\login.php(2): include()
#2 D:\xammp\htdocs\ci4\system\View\View.php(210): include('D:\\xammp\\htdocs...')
#3 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login', Array, true)
#5 D:\xammp\htdocs\ci4\app\Controllers\Login.php(10): view('login')
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Login->index()
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-02-16 11:00:11 --> include(include/head.php): failed to open stream: No such file or directory
#0 D:\xammp\htdocs\ci4\app\Views\login.php(2): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(include...', 'D:\\xammp\\htdocs...', 2, Array)
#1 D:\xammp\htdocs\ci4\app\Views\login.php(2): include()
#2 D:\xammp\htdocs\ci4\system\View\View.php(210): include('D:\\xammp\\htdocs...')
#3 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login', Array, true)
#5 D:\xammp\htdocs\ci4\app\Controllers\Login.php(10): view('login')
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Login->index()
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-02-16 11:00:16 --> include(include/head.php): failed to open stream: No such file or directory
#0 D:\xammp\htdocs\ci4\app\Views\login.php(2): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(include...', 'D:\\xammp\\htdocs...', 2, Array)
#1 D:\xammp\htdocs\ci4\app\Views\login.php(2): include()
#2 D:\xammp\htdocs\ci4\system\View\View.php(210): include('D:\\xammp\\htdocs...')
#3 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login', Array, true)
#5 D:\xammp\htdocs\ci4\app\Controllers\Login.php(10): view('login')
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Login->index()
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-02-16 11:00:53 --> include(include/head.php): failed to open stream: No such file or directory
#0 D:\xammp\htdocs\ci4\app\Views\login.php(2): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(include...', 'D:\\xammp\\htdocs...', 2, Array)
#1 D:\xammp\htdocs\ci4\app\Views\login.php(2): include()
#2 D:\xammp\htdocs\ci4\system\View\View.php(210): include('D:\\xammp\\htdocs...')
#3 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login.php', Array, true)
#5 D:\xammp\htdocs\ci4\app\Controllers\Login.php(10): view('login.php')
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Login->index()
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-02-16 11:01:52 --> include(include/head.php): failed to open stream: No such file or directory
#0 D:\xammp\htdocs\ci4\app\Views\login.php(2): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(include...', 'D:\\xammp\\htdocs...', 2, Array)
#1 D:\xammp\htdocs\ci4\app\Views\login.php(2): include()
#2 D:\xammp\htdocs\ci4\system\View\View.php(210): include('D:\\xammp\\htdocs...')
#3 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login.php', Array, true)
#5 D:\xammp\htdocs\ci4\app\Controllers\Login.php(10): view('login.php')
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Login->index()
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-02-16 11:02:19 --> include(include/head.php): failed to open stream: No such file or directory
#0 D:\xammp\htdocs\ci4\app\Views\loginn.php(2): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(include...', 'D:\\xammp\\htdocs...', 2, Array)
#1 D:\xammp\htdocs\ci4\app\Views\loginn.php(2): include()
#2 D:\xammp\htdocs\ci4\system\View\View.php(210): include('D:\\xammp\\htdocs...')
#3 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('loginn', Array, true)
#5 D:\xammp\htdocs\ci4\app\Controllers\Login.php(10): view('loginn')
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Login->index()
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-02-16 11:02:20 --> include(include/head.php): failed to open stream: No such file or directory
#0 D:\xammp\htdocs\ci4\app\Views\loginn.php(2): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(include...', 'D:\\xammp\\htdocs...', 2, Array)
#1 D:\xammp\htdocs\ci4\app\Views\loginn.php(2): include()
#2 D:\xammp\htdocs\ci4\system\View\View.php(210): include('D:\\xammp\\htdocs...')
#3 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('loginn', Array, true)
#5 D:\xammp\htdocs\ci4\app\Controllers\Login.php(10): view('loginn')
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Login->index()
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-02-16 11:02:21 --> include(include/head.php): failed to open stream: No such file or directory
#0 D:\xammp\htdocs\ci4\app\Views\loginn.php(2): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(include...', 'D:\\xammp\\htdocs...', 2, Array)
#1 D:\xammp\htdocs\ci4\app\Views\loginn.php(2): include()
#2 D:\xammp\htdocs\ci4\system\View\View.php(210): include('D:\\xammp\\htdocs...')
#3 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('loginn', Array, true)
#5 D:\xammp\htdocs\ci4\app\Controllers\Login.php(10): view('loginn')
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Login->index()
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-02-16 11:02:35 --> Invalid file: login.php
#0 D:\xammp\htdocs\ci4\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('login.php')
#1 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login', Array, true)
#2 D:\xammp\htdocs\ci4\app\Controllers\Login.php(10): view('login')
#3 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Login->index()
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-02-16 11:02:36 --> Invalid file: login.php
#0 D:\xammp\htdocs\ci4\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('login.php')
#1 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login', Array, true)
#2 D:\xammp\htdocs\ci4\app\Controllers\Login.php(10): view('login')
#3 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Login->index()
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-02-16 11:02:38 --> Invalid file: login.php
#0 D:\xammp\htdocs\ci4\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('login.php')
#1 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login', Array, true)
#2 D:\xammp\htdocs\ci4\app\Controllers\Login.php(10): view('login')
#3 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Login->index()
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-02-16 11:04:08 --> Invalid file: login.php
#0 D:\xammp\htdocs\ci4\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('login.php')
#1 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login', Array, true)
#2 D:\xammp\htdocs\ci4\app\Controllers\Login.php(10): view('login')
#3 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Login->index()
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-02-16 11:04:09 --> Invalid file: login.php
#0 D:\xammp\htdocs\ci4\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('login.php')
#1 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login', Array, true)
#2 D:\xammp\htdocs\ci4\app\Controllers\Login.php(10): view('login')
#3 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Login->index()
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-02-16 11:05:24 --> Invalid file: login.php
#0 D:\xammp\htdocs\ci4\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('login.php')
#1 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login', Array, true)
#2 D:\xammp\htdocs\ci4\app\Controllers\Login.php(10): view('login')
#3 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Login->index()
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-02-16 11:05:32 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Router\Router.php(175): CodeIgniter\Router\Router->autoRoute('/registration')
#1 D:\xammp\htdocs\ci4\system\CodeIgniter.php(719): CodeIgniter\Router\Router->handle('/registration')
#2 D:\xammp\htdocs\ci4\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#3 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-02-16 11:06:04 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Router\Router.php(175): CodeIgniter\Router\Router->autoRoute('/registration')
#1 D:\xammp\htdocs\ci4\system\CodeIgniter.php(719): CodeIgniter\Router\Router->handle('/registration')
#2 D:\xammp\htdocs\ci4\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#3 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-02-16 11:06:09 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Router\Router.php(175): CodeIgniter\Router\Router->autoRoute('/registration')
#1 D:\xammp\htdocs\ci4\system\CodeIgniter.php(719): CodeIgniter\Router\Router->handle('/registration')
#2 D:\xammp\htdocs\ci4\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#3 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-02-16 11:07:33 --> syntax error, unexpected 'if' (T_IF)
#0 D:\xammp\htdocs\ci4\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->tryToRouteIt(NULL)
#1 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#2 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#3 {main}
CRITICAL - 2022-02-16 11:07:37 --> syntax error, unexpected 'if' (T_IF)
#0 D:\xammp\htdocs\ci4\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->tryToRouteIt(NULL)
#1 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#2 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#3 {main}
CRITICAL - 2022-02-16 11:16:03 --> syntax error, unexpected 'if' (T_IF)
#0 D:\xammp\htdocs\ci4\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->tryToRouteIt(NULL)
#1 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#2 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#3 {main}
CRITICAL - 2022-02-16 11:16:16 --> Invalid file: login.php
#0 D:\xammp\htdocs\ci4\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('login.php')
#1 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login', Array, true)
#2 D:\xammp\htdocs\ci4\app\Controllers\Login.php(10): view('login')
#3 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Login->index()
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-02-16 11:16:21 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Router\Router.php(175): CodeIgniter\Router\Router->autoRoute('/registration')
#1 D:\xammp\htdocs\ci4\system\CodeIgniter.php(719): CodeIgniter\Router\Router->handle('/registration')
#2 D:\xammp\htdocs\ci4\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#3 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-02-16 11:19:27 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Router\Router.php(175): CodeIgniter\Router\Router->autoRoute('/registration')
#1 D:\xammp\htdocs\ci4\system\CodeIgniter.php(719): CodeIgniter\Router\Router->handle('/registration')
#2 D:\xammp\htdocs\ci4\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#3 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-02-16 11:19:38 --> Invalid file: login.php
#0 D:\xammp\htdocs\ci4\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('login.php')
#1 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login', Array, true)
#2 D:\xammp\htdocs\ci4\app\Controllers\Login.php(10): view('login')
#3 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Login->index()
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-02-16 11:20:24 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 11:20:30 --> Invalid file: login.php
#0 D:\xammp\htdocs\ci4\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('login.php')
#1 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login', Array, true)
#2 D:\xammp\htdocs\ci4\app\Controllers\Login.php(10): view('login')
#3 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Login->index()
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-02-16 11:21:04 --> Invalid file: login.php
#0 D:\xammp\htdocs\ci4\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('login.php')
#1 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login', Array, true)
#2 D:\xammp\htdocs\ci4\app\Controllers\Login.php(10): view('login')
#3 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Login->index()
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-02-16 11:21:10 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 11:21:42 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 11:23:05 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 11:23:14 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 11:23:33 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 11:23:46 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 11:23:47 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 11:23:51 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
INFO - 2022-02-16 11:24:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2022-02-16 11:25:22 --> Invalid file: login.php
#0 D:\xammp\htdocs\ci4\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('login.php')
#1 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login', Array, true)
#2 D:\xammp\htdocs\ci4\app\Config\Routes.php(40): view('login')
#3 D:\xammp\htdocs\ci4\system\CodeIgniter.php(781): CodeIgniter\CodeIgniter->Config\{closure}()
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
INFO - 2022-02-16 11:25:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2022-02-16 11:25:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2022-02-16 11:25:29 --> Invalid file: login.php
#0 D:\xammp\htdocs\ci4\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('login.php')
#1 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login', Array, true)
#2 D:\xammp\htdocs\ci4\app\Config\Routes.php(40): view('login')
#3 D:\xammp\htdocs\ci4\system\CodeIgniter.php(781): CodeIgniter\CodeIgniter->Config\{closure}()
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-02-16 11:26:03 --> Invalid file: login.php
#0 D:\xammp\htdocs\ci4\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('login.php')
#1 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login', Array, true)
#2 D:\xammp\htdocs\ci4\app\Config\Routes.php(40): view('login')
#3 D:\xammp\htdocs\ci4\system\CodeIgniter.php(781): CodeIgniter\CodeIgniter->Config\{closure}()
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-02-16 11:26:04 --> Invalid file: login.php
#0 D:\xammp\htdocs\ci4\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('login.php')
#1 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login', Array, true)
#2 D:\xammp\htdocs\ci4\app\Config\Routes.php(40): view('login')
#3 D:\xammp\htdocs\ci4\system\CodeIgniter.php(781): CodeIgniter\CodeIgniter->Config\{closure}()
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
INFO - 2022-02-16 11:26:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2022-02-16 11:26:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2022-02-16 11:26:12 --> Invalid file: login.php
#0 D:\xammp\htdocs\ci4\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('login.php')
#1 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login', Array, true)
#2 D:\xammp\htdocs\ci4\app\Config\Routes.php(40): view('login')
#3 D:\xammp\htdocs\ci4\system\CodeIgniter.php(781): CodeIgniter\CodeIgniter->Config\{closure}()
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
INFO - 2022-02-16 11:26:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2022-02-16 11:27:07 --> include(include/head.php): failed to open stream: No such file or directory
#0 D:\xammp\htdocs\ci4\app\Views\loginn.php(2): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(include...', 'D:\\xammp\\htdocs...', 2, Array)
#1 D:\xammp\htdocs\ci4\app\Views\loginn.php(2): include()
#2 D:\xammp\htdocs\ci4\system\View\View.php(210): include('D:\\xammp\\htdocs...')
#3 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('loginn', Array, true)
#5 D:\xammp\htdocs\ci4\app\Config\Routes.php(40): view('loginn')
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(781): CodeIgniter\CodeIgniter->Config\{closure}()
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-02-16 11:27:10 --> include(include/head.php): failed to open stream: No such file or directory
#0 D:\xammp\htdocs\ci4\app\Views\loginn.php(2): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(include...', 'D:\\xammp\\htdocs...', 2, Array)
#1 D:\xammp\htdocs\ci4\app\Views\loginn.php(2): include()
#2 D:\xammp\htdocs\ci4\system\View\View.php(210): include('D:\\xammp\\htdocs...')
#3 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('loginn', Array, true)
#5 D:\xammp\htdocs\ci4\app\Config\Routes.php(40): view('loginn')
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(781): CodeIgniter\CodeIgniter->Config\{closure}()
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
INFO - 2022-02-16 11:27:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2022-02-16 11:27:14 --> include(include/head.php): failed to open stream: No such file or directory
#0 D:\xammp\htdocs\ci4\app\Views\loginn.php(2): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(include...', 'D:\\xammp\\htdocs...', 2, Array)
#1 D:\xammp\htdocs\ci4\app\Views\loginn.php(2): include()
#2 D:\xammp\htdocs\ci4\system\View\View.php(210): include('D:\\xammp\\htdocs...')
#3 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('loginn', Array, true)
#5 D:\xammp\htdocs\ci4\app\Config\Routes.php(40): view('loginn')
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(781): CodeIgniter\CodeIgniter->Config\{closure}()
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
INFO - 2022-02-16 11:27:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2022-02-16 11:27:32 --> include(include/head.php): failed to open stream: No such file or directory
#0 D:\xammp\htdocs\ci4\app\Views\loginn.php(2): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(include...', 'D:\\xammp\\htdocs...', 2, Array)
#1 D:\xammp\htdocs\ci4\app\Views\loginn.php(2): include()
#2 D:\xammp\htdocs\ci4\system\View\View.php(210): include('D:\\xammp\\htdocs...')
#3 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('loginn', Array, true)
#5 D:\xammp\htdocs\ci4\app\Config\Routes.php(41): view('loginn')
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(781): CodeIgniter\CodeIgniter->Config\{closure}()
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-02-16 11:28:28 --> include(include/head.php): failed to open stream: No such file or directory
#0 D:\xammp\htdocs\ci4\app\Views\login_form.php(2): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(include...', 'D:\\xammp\\htdocs...', 2, Array)
#1 D:\xammp\htdocs\ci4\app\Views\login_form.php(2): include()
#2 D:\xammp\htdocs\ci4\system\View\View.php(210): include('D:\\xammp\\htdocs...')
#3 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login_form', Array, true)
#5 D:\xammp\htdocs\ci4\app\Config\Routes.php(40): view('login_form')
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(781): CodeIgniter\CodeIgniter->Config\{closure}()
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-02-16 11:28:29 --> include(include/head.php): failed to open stream: No such file or directory
#0 D:\xammp\htdocs\ci4\app\Views\login_form.php(2): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(include...', 'D:\\xammp\\htdocs...', 2, Array)
#1 D:\xammp\htdocs\ci4\app\Views\login_form.php(2): include()
#2 D:\xammp\htdocs\ci4\system\View\View.php(210): include('D:\\xammp\\htdocs...')
#3 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login_form', Array, true)
#5 D:\xammp\htdocs\ci4\app\Config\Routes.php(40): view('login_form')
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(781): CodeIgniter\CodeIgniter->Config\{closure}()
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
INFO - 2022-02-16 11:28:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2022-02-16 11:28:35 --> include(include/head.php): failed to open stream: No such file or directory
#0 D:\xammp\htdocs\ci4\app\Views\login_form.php(2): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(include...', 'D:\\xammp\\htdocs...', 2, Array)
#1 D:\xammp\htdocs\ci4\app\Views\login_form.php(2): include()
#2 D:\xammp\htdocs\ci4\system\View\View.php(210): include('D:\\xammp\\htdocs...')
#3 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login_form', Array, true)
#5 D:\xammp\htdocs\ci4\app\Config\Routes.php(40): view('login_form')
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(781): CodeIgniter\CodeIgniter->Config\{closure}()
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
INFO - 2022-02-16 11:28:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2022-02-16 11:29:02 --> include(include/head.php): failed to open stream: No such file or directory
#0 D:\xammp\htdocs\ci4\app\Views\login_form.php(2): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(include...', 'D:\\xammp\\htdocs...', 2, Array)
#1 D:\xammp\htdocs\ci4\app\Views\login_form.php(2): include()
#2 D:\xammp\htdocs\ci4\system\View\View.php(210): include('D:\\xammp\\htdocs...')
#3 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login_form', Array, true)
#5 D:\xammp\htdocs\ci4\app\Config\Routes.php(40): view('login_form')
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(781): CodeIgniter\CodeIgniter->Config\{closure}()
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-02-16 12:04:48 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
INFO - 2022-02-16 12:07:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2022-02-16 12:08:28 --> include(include/head.php): failed to open stream: No such file or directory
#0 D:\xammp\htdocs\ci4\app\Views\login_form.php(2): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(include...', 'D:\\xammp\\htdocs...', 2, Array)
#1 D:\xammp\htdocs\ci4\app\Views\login_form.php(2): include()
#2 D:\xammp\htdocs\ci4\system\View\View.php(210): include('D:\\xammp\\htdocs...')
#3 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login_form', Array, true)
#5 D:\xammp\htdocs\ci4\app\Config\Routes.php(42): view('login_form')
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(781): CodeIgniter\CodeIgniter->Config\{closure}()
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
INFO - 2022-02-16 12:08:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2022-02-16 12:09:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2022-02-16 12:09:52 --> include(include/head.php): failed to open stream: No such file or directory
#0 D:\xammp\htdocs\ci4\app\Views\login_form.php(3): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(include...', 'D:\\xammp\\htdocs...', 3, Array)
#1 D:\xammp\htdocs\ci4\app\Views\login_form.php(3): include()
#2 D:\xammp\htdocs\ci4\system\View\View.php(210): include('D:\\xammp\\htdocs...')
#3 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login_form', Array, true)
#5 D:\xammp\htdocs\ci4\app\Config\Routes.php(42): view('login_form')
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(781): CodeIgniter\CodeIgniter->Config\{closure}()
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-02-16 12:10:35 --> include(include/head.php): failed to open stream: No such file or directory
#0 D:\xammp\htdocs\ci4\app\Views\login_form.php(3): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(include...', 'D:\\xammp\\htdocs...', 3, Array)
#1 D:\xammp\htdocs\ci4\app\Views\login_form.php(3): include()
#2 D:\xammp\htdocs\ci4\system\View\View.php(210): include('D:\\xammp\\htdocs...')
#3 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login_form', Array, true)
#5 D:\xammp\htdocs\ci4\app\Config\Routes.php(42): view('login_form')
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(781): CodeIgniter\CodeIgniter->Config\{closure}()
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-02-16 12:10:45 --> include(include/head.php): failed to open stream: No such file or directory
#0 D:\xammp\htdocs\ci4\app\Views\login_form.php(3): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(include...', 'D:\\xammp\\htdocs...', 3, Array)
#1 D:\xammp\htdocs\ci4\app\Views\login_form.php(3): include()
#2 D:\xammp\htdocs\ci4\system\View\View.php(210): include('D:\\xammp\\htdocs...')
#3 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login_form', Array, true)
#5 D:\xammp\htdocs\ci4\app\Config\Routes.php(42): view('login_form')
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(781): CodeIgniter\CodeIgniter->Config\{closure}()
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
INFO - 2022-02-16 12:10:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2022-02-16 12:10:53 --> include(include/head.php): failed to open stream: No such file or directory
#0 D:\xammp\htdocs\ci4\app\Views\login_form.php(3): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(include...', 'D:\\xammp\\htdocs...', 3, Array)
#1 D:\xammp\htdocs\ci4\app\Views\login_form.php(3): include()
#2 D:\xammp\htdocs\ci4\system\View\View.php(210): include('D:\\xammp\\htdocs...')
#3 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login_form', Array, true)
#5 D:\xammp\htdocs\ci4\app\Config\Routes.php(42): view('login_form')
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(781): CodeIgniter\CodeIgniter->Config\{closure}()
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-02-16 12:11:14 --> include(include/head.php): failed to open stream: No such file or directory
#0 D:\xammp\htdocs\ci4\app\Views\login_form.php(3): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(include...', 'D:\\xammp\\htdocs...', 3, Array)
#1 D:\xammp\htdocs\ci4\app\Views\login_form.php(3): include()
#2 D:\xammp\htdocs\ci4\system\View\View.php(210): include('D:\\xammp\\htdocs...')
#3 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login_form', Array, true)
#5 D:\xammp\htdocs\ci4\app\Config\Routes.php(42): view('login_form')
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(781): CodeIgniter\CodeIgniter->Config\{closure}()
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-02-16 12:11:29 --> include(include/head.php): failed to open stream: No such file or directory
#0 D:\xammp\htdocs\ci4\app\Views\login_form.php(3): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(include...', 'D:\\xammp\\htdocs...', 3, Array)
#1 D:\xammp\htdocs\ci4\app\Views\login_form.php(3): include()
#2 D:\xammp\htdocs\ci4\system\View\View.php(210): include('D:\\xammp\\htdocs...')
#3 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#4 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('login_form', Array, true)
#5 D:\xammp\htdocs\ci4\app\Config\Routes.php(42): view('login_form')
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(781): CodeIgniter\CodeIgniter->Config\{closure}()
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
INFO - 2022-02-16 12:12:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2022-02-16 12:12:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2022-02-16 12:12:25 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
INFO - 2022-02-16 12:14:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2022-02-16 12:14:59 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
INFO - 2022-02-16 12:15:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2022-02-16 12:18:00 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
INFO - 2022-02-16 12:18:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2022-02-16 12:18:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2022-02-16 12:18:09 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
INFO - 2022-02-16 12:19:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2022-02-16 12:21:59 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
INFO - 2022-02-16 12:22:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2022-02-16 12:22:09 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
INFO - 2022-02-16 12:22:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2022-02-16 12:23:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2022-02-16 12:23:07 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
INFO - 2022-02-16 12:23:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2022-02-16 12:24:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2022-02-16 12:24:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2022-02-16 12:24:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2022-02-16 12:24:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2022-02-16 12:25:49 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 12:26:07 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 12:26:09 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 12:28:22 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 12:28:26 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 12:30:09 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 12:30:13 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 12:30:37 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 12:34:17 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
INFO - 2022-02-16 12:34:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2022-02-16 12:34:31 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
INFO - 2022-02-16 12:34:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2022-02-16 12:35:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2022-02-16 12:36:06 --> syntax error, unexpected ')', expecting ']'
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 12:36:56 --> syntax error, unexpected '(', expecting variable (T_VARIABLE)
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
INFO - 2022-02-16 12:37:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2022-02-16 12:37:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2022-02-16 12:37:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2022-02-16 12:37:40 --> Call to undefined function App\Controllers\validate()
#0 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->registerCustomer()
#1 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#2 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2022-02-16 12:37:44 --> Call to undefined function App\Controllers\validate()
#0 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->registerCustomer()
#1 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#2 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
INFO - 2022-02-16 12:40:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2022-02-16 12:40:09 --> Call to undefined function display_error()
#0 D:\xammp\htdocs\ci4\system\View\View.php(210): include()
#1 D:\xammp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xammp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('registration', Array, true)
#3 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(62): view('registration', Array)
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->registerCustomer()
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
INFO - 2022-02-16 12:40:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2022-02-16 12:41:45 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-02-16 12:41:45 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 D:\xammp\htdocs\ci4\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\xammp\htdocs\ci4\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 D:\xammp\htdocs\ci4\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 D:\xammp\htdocs\ci4\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('m.adnan6425@gma...', 'cust_email', Array, NULL)
#4 D:\xammp\htdocs\ci4\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'm.adnan6425@gma...', Array, Array)
#5 D:\xammp\htdocs\ci4\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(59): CodeIgniter\Controller->validate(Array)
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->registerCustomer()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#9 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
INFO - 2022-02-16 12:44:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2022-02-16 12:44:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2022-02-16 12:45:01 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-02-16 12:45:01 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 D:\xammp\htdocs\ci4\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\xammp\htdocs\ci4\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 D:\xammp\htdocs\ci4\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 D:\xammp\htdocs\ci4\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('wy5on@gc9c.com', 'cust_email', Array, NULL)
#4 D:\xammp\htdocs\ci4\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'wy5on@gc9c.com', Array, Array)
#5 D:\xammp\htdocs\ci4\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(59): CodeIgniter\Controller->validate(Array)
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->registerCustomer()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#9 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
INFO - 2022-02-16 12:45:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2022-02-16 12:45:57 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-02-16 12:45:57 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 D:\xammp\htdocs\ci4\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\xammp\htdocs\ci4\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 D:\xammp\htdocs\ci4\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 D:\xammp\htdocs\ci4\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('m.adnan6425@gma...', 'cust_email', Array, NULL)
#4 D:\xammp\htdocs\ci4\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'm.adnan6425@gma...', Array, Array)
#5 D:\xammp\htdocs\ci4\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(59): CodeIgniter\Controller->validate(Array)
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->registerCustomer()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#9 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-16 12:47:15 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-02-16 12:47:15 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 D:\xammp\htdocs\ci4\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\xammp\htdocs\ci4\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 D:\xammp\htdocs\ci4\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 D:\xammp\htdocs\ci4\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('m.adnan6425@gma...', 'cust_email', Array, NULL)
#4 D:\xammp\htdocs\ci4\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'm.adnan6425@gma...', Array, Array)
#5 D:\xammp\htdocs\ci4\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(59): CodeIgniter\Controller->validate(Array)
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->registerCustomer()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#9 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
INFO - 2022-02-16 12:47:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2022-02-16 12:47:30 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-02-16 12:47:30 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 D:\xammp\htdocs\ci4\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\xammp\htdocs\ci4\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 D:\xammp\htdocs\ci4\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 D:\xammp\htdocs\ci4\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('m.adnan6425@gma...', 'cust_email', Array, NULL)
#4 D:\xammp\htdocs\ci4\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'm.adnan6425@gma...', Array, Array)
#5 D:\xammp\htdocs\ci4\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(59): CodeIgniter\Controller->validate(Array)
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->registerCustomer()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#9 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2022-02-16 12:53:10 --> syntax error, unexpected '}', expecting ';' or ','
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-02-16 12:53:14 --> syntax error, unexpected '}', expecting ';' or ','
#0 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xammp\\htdocs...')
#1 D:\xammp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xammp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 D:\xammp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
INFO - 2022-02-16 12:54:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2022-02-16 12:54:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2022-02-16 12:55:02 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-02-16 12:55:02 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 D:\xammp\htdocs\ci4\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\xammp\htdocs\ci4\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 D:\xammp\htdocs\ci4\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 D:\xammp\htdocs\ci4\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('m.adnan6425@gma...', 'cust_email', Array, NULL)
#4 D:\xammp\htdocs\ci4\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'm.adnan6425@gma...', Array, Array)
#5 D:\xammp\htdocs\ci4\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(59): CodeIgniter\Controller->validate(Array)
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->registerCustomer()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#9 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-16 12:56:54 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-02-16 12:56:54 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 D:\xammp\htdocs\ci4\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\xammp\htdocs\ci4\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 D:\xammp\htdocs\ci4\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 D:\xammp\htdocs\ci4\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('m.adnan6425@gma...', 'cust_email', Array, NULL)
#4 D:\xammp\htdocs\ci4\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'm.adnan6425@gma...', Array, Array)
#5 D:\xammp\htdocs\ci4\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(59): CodeIgniter\Controller->validate(Array)
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->registerCustomer()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#9 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
INFO - 2022-02-16 12:56:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2022-02-16 12:57:05 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-02-16 12:57:05 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 D:\xammp\htdocs\ci4\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\xammp\htdocs\ci4\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 D:\xammp\htdocs\ci4\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 D:\xammp\htdocs\ci4\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('m.adnan6425@gma...', 'cust_email', Array, NULL)
#4 D:\xammp\htdocs\ci4\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'm.adnan6425@gma...', Array, Array)
#5 D:\xammp\htdocs\ci4\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(59): CodeIgniter\Controller->validate(Array)
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->registerCustomer()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#9 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-16 12:57:23 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-02-16 12:57:23 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 D:\xammp\htdocs\ci4\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\xammp\htdocs\ci4\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 D:\xammp\htdocs\ci4\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 D:\xammp\htdocs\ci4\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('m.adnan6425@gma...', 'cust_email', Array, NULL)
#4 D:\xammp\htdocs\ci4\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'm.adnan6425@gma...', Array, Array)
#5 D:\xammp\htdocs\ci4\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(59): CodeIgniter\Controller->validate(Array)
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->registerCustomer()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#9 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
INFO - 2022-02-16 12:57:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2022-02-16 12:57:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2022-02-16 12:57:34 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-02-16 12:57:34 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 D:\xammp\htdocs\ci4\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\xammp\htdocs\ci4\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 D:\xammp\htdocs\ci4\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 D:\xammp\htdocs\ci4\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('m.adnan6425@gma...', 'cust_email', Array, NULL)
#4 D:\xammp\htdocs\ci4\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'm.adnan6425@gma...', Array, Array)
#5 D:\xammp\htdocs\ci4\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(59): CodeIgniter\Controller->validate(Array)
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->registerCustomer()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#9 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-16 12:59:08 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-02-16 12:59:08 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 D:\xammp\htdocs\ci4\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\xammp\htdocs\ci4\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 D:\xammp\htdocs\ci4\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 D:\xammp\htdocs\ci4\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('m.adnan6425@gma...', 'cust_email', Array, NULL)
#4 D:\xammp\htdocs\ci4\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'm.adnan6425@gma...', Array, Array)
#5 D:\xammp\htdocs\ci4\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(60): CodeIgniter\Controller->validate(Array)
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->registerCustomer()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#9 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
INFO - 2022-02-16 12:59:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2022-02-16 12:59:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2022-02-16 12:59:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2022-02-16 12:59:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2022-02-16 12:59:27 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-02-16 12:59:27 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 D:\xammp\htdocs\ci4\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\xammp\htdocs\ci4\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 D:\xammp\htdocs\ci4\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 D:\xammp\htdocs\ci4\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('m.adnan6425@gma...', 'cust_email', Array, NULL)
#4 D:\xammp\htdocs\ci4\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'm.adnan6425@gma...', Array, Array)
#5 D:\xammp\htdocs\ci4\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(60): CodeIgniter\Controller->validate(Array)
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->registerCustomer()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#9 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-16 13:00:22 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-02-16 13:00:22 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 D:\xammp\htdocs\ci4\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\xammp\htdocs\ci4\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 D:\xammp\htdocs\ci4\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 D:\xammp\htdocs\ci4\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('m.adnan6425@gma...', 'cust_email', Array, NULL)
#4 D:\xammp\htdocs\ci4\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'm.adnan6425@gma...', Array, Array)
#5 D:\xammp\htdocs\ci4\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(62): CodeIgniter\Controller->validate(Array)
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->registerCustomer()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#9 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
INFO - 2022-02-16 13:00:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2022-02-16 13:00:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2022-02-16 13:01:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2022-02-16 13:01:32 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-02-16 13:01:32 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 D:\xammp\htdocs\ci4\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\xammp\htdocs\ci4\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 D:\xammp\htdocs\ci4\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 D:\xammp\htdocs\ci4\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('m.adnan6425@gma...', 'cust_email', Array, NULL)
#4 D:\xammp\htdocs\ci4\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'm.adnan6425@gma...', Array, Array)
#5 D:\xammp\htdocs\ci4\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(62): CodeIgniter\Controller->validate(Array)
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->registerCustomer()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#9 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-16 13:04:07 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-02-16 13:04:07 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 D:\xammp\htdocs\ci4\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\xammp\htdocs\ci4\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 D:\xammp\htdocs\ci4\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 D:\xammp\htdocs\ci4\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('m.adnan6425@gma...', 'cust_email', Array, NULL)
#4 D:\xammp\htdocs\ci4\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'm.adnan6425@gma...', Array, Array)
#5 D:\xammp\htdocs\ci4\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(62): CodeIgniter\Controller->validate(Array)
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->registerCustomer()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#9 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2022-02-16 13:04:10 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-02-16 13:04:10 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 D:\xammp\htdocs\ci4\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\xammp\htdocs\ci4\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#2 D:\xammp\htdocs\ci4\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#3 D:\xammp\htdocs\ci4\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('m.adnan6425@gma...', 'cust_email', Array, NULL)
#4 D:\xammp\htdocs\ci4\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'm.adnan6425@gma...', Array, Array)
#5 D:\xammp\htdocs\ci4\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#6 D:\xammp\htdocs\ci4\app\Controllers\Registration.php(62): CodeIgniter\Controller->validate(Array)
#7 D:\xammp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\Registration->registerCustomer()
#8 D:\xammp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registration))
#9 D:\xammp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xammp\htdocs\ci4\index.php(37): CodeIgniter\CodeIgniter->run()
#11 {main}
INFO - 2022-02-16 13:04:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2022-02-16 13:04:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
